nms/app/packages/magmalte/app/components/GatewayCommandFields.js

```diff
@@ -466,24 +468,41 @@ export function TroubleshootingControl(props: ChildProps) {
     controlProxyContent,
     setControlProxyContent,
   ] = useState<generic_command_response>({});
+  const [
+    tdAgentLogsContent,
+    setTdAgentLogsContent,
+  ] = useState<generic_command_response>({});
   const networkId = nullthrows(match.params.networkId);
-  const parameters = {
+  const controlProxyParams = {
     command: 'bash',
     params: {
-      shell_params: ["-c 'cat /etc/magma/control_proxy.yml'"],
+      shell_params: ["-c 'cat /var/opt/magma/configs/control_proxy.yml'"],
```

Comment:
I know this is probably the only way to do it, but do you think there's a way we can get this to work without having to know the exact filepath of this config file?

At the very least it would be nice to pull this (or just the directory path) out into a constant.

Reply:
Yeah. Makes sense. I will make it as a constant.